ci/Jenkinsfile

```diff
@@ -2,7 +2,7 @@ node("docsbuilder-maven") {
   checkout scm
   stage("Build Documentation") {
     sh "pwd && ls -l"
-    sh "echo :revnumber: \$(git rev-parse --short=10 HEAD) >> ./docs/topics/templates/document-attributes.adoc"
+    sh "echo :revnumber: \$(date --rfc-3339=date) >> ./docs/topics/templates/document-attributes.adoc"
```

Comment:
I think this is ultimately @rhoads-zach's call, but having the ISO date sorta solves #288, as you said yourself. I'm ambivalent towards this.